api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -579,6 +580,12 @@ private boolean userHasEverLoggedIn(
     return ResponseEntity.ok(new EmptyResponse());
   }
 
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  public ResponseEntity<Profile> editUserInformation(EditUserInformationRequest request) {
+    return ResponseEntity.ok(profileService.editUserInformation(request));
```

Comment:
I'm undecided on whether to move `ACCESS_CONTROL_ADMIN` endpoints like these to a new FooAdminController/Service.  Certainly this will share the fate of others like it, whatever we decide on.  Before then, these will get pushed down to ProfileService/Impl like the others.

Reply:
I prefer a separate `UserAdminController`, leaving the profile stuff to just user-editable fields. Otherwise it's going to be too easy to accidentally give non-administrators access via some random codepath. I've already got changes to do that on one of my branches.

Reply:
I'm a bit concerned that "user information" is not a well-scoped abstraction. What _isn't_ user information? How about `updateAccountAdminProperties()` or something?

Reply:
"User Information" comes from the UI control.  I agree that it's vague and should be changed.  Something like AdminAccountProperties, yeah.  Let's see how that looks.